### ISSUE TITLE
PTN001501163 - colocalizes_with ionotropic glutamate receptor complex

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-annotation#2382]
PTN001501163:
please remove:
colocalizes_with ionotropic glutamate receptor complex - we've agreed not to make colocalizes_with with complex terms

PTN001501163 - there are a lot of annotations from this node. Consider removing tems that are only supported by a single annotation:
establishment or maintenance of epithelial cell apical/basal polarity
receptor clustering
receptor localization to synapse

**Comment by marcfeuermann** [comment]
Hi Helen,
colocalizes_with ionotropic glutamate receptor complex has been removed. 

I do not understand your other requests:
"receptor localization to synapse" propagation results from 6 primary annotations: 
directly from Rat Dlg4, but is also parent of "neurotransmitter receptor localization to postsynaptic specialization membrane" (Mouse Dlg1, Rat Dlg1, Mouse Dlg2 and mouse Dlg4), and of "neurotransmitter receptor transport postsynaptic membrane to endosome" (Rat_Scrib). Mouse Dlg3 has an annotation to "regulation of postsynaptic membrane neurotransmitter receptor levels", which, even if not a child, points to the same direction.
So we have many evidences to be confident to propagate this term. The same is true for most of the other terms which are also supported by several evidences (even if not always direct).
Moreover, I think that a single annotation is sometimes enough to allow a propagation (we can have other evidences from literature, protein interactions, ...). 

So I'm not convinced that not propagating terms that are only supported by a single annotation should be a general rule for PAINT. This should probably be further discussed. @pgaudet, @huaiyumi, @ValWood, some comments ?

**Comment by ValWood** [comment]
I don't know anything about this specific  area of biology, but  I think sometimes very good evidence from  a single source can often be propagated. For example, if an unknown gene is (well) characterized in yeast,  I would recommend broad propagation.

I would say that each annotation/propagation  would need to be decided in context (taking into account copy number and taxonomic distribution), which is what @marcfeuermann does. No one size fits all....

**Comment by pgaudet** [comment]
> So I'm not convinced that not propagating terms that are only supported by a single annotation should be a general rule for PAINT.

I agree. I know Helen you are worried about those cases, but perhaps you unfortunately encountered cases where these propagations were dubious. I am quite sure there are many more cases where the annotations are correct and propagation appropriate. 

It's not always the case that research in reproduced in multiple organisms. 

Pascale

**Comment by hattrill** [comment]
On the subject of single evidence propagations:

I think there needs to be a stricter rule for propagating with a single gene supporting evidence, as for the bad examples I have encountered, there has not always been clear reason as to why it was chosen as a general, propagatable property.

There is also the issue of when a single source is used, that sometimes it pops up as an IBA for the source, making it completely circular.

Here are a number of examples where I've had issues with single source IBAs:
#2219
#2525
#2385
#2381
#2339

I haven't done a comprehensive survey to find out how big the problem is, these are just examples I come across as part of focussed projects.

I understand that there will be examples when there is a good reason to propagate an annotation from a single source e.g. from a well-conserved ribosome subunit the curator can be fairly certain that that propagation of terms related directly to ribosome function are correct. But the problem arises when the terms are looser, related to a broader biological function such as e.g. neuron differentiation or a term that might be just a result of how the curator has interpreted the experiment e.g. receptor localization to synapse.

IMO, sticking to the rule: must be >1 supporting plus clear process-function link solves all this (and although some good stuff doesn't get propagated the bigger win is that harmful annotations don't).

At very minimum, clearer rules in an SOP (visible to all), such as there must be a strong and clear link between the MF/CC and BP should underlie single source IBAs (which should be a rare event):
e.g.  a CC: cullin-RING ubiquitin ligase complex or MF:ubiquitin protein ligase activity
therefore is safe to propagate:
protein ubiquitination
but not safe to propagate unrelated BPs which may be context specific or wrong (ideally true for all PAINT propagations, but particularly with single evidence)
e.g.
MF:ubiquitin protein ligase activity
plus:
dsRNA transport
negative regulation of epidermal growth factor receptor signaling pathway

Although some curators may have this rule in their head, it is clear that there are annotations for which this is not true.

**Comment by pgaudet** [comment]
Good point about the guidelines. I'll add that to our next PAINT call.

**Comment by ValWood** [comment]
yep. I'm sure it gets better over time, and you are right about   things that might be over-propagation (curator focussing on phenotype rather than process). At least we get better.
You might be able to use Matrix to weed out these in the development and signalling branches, but we would need to think of suitable  violating intersections. Maybe based on function and process. There must be examples where, if you dont know the the MF you cannot possibly be  very specific about a process from a phenotype. 
We could look at your tickets sometime and see if we can figure out any useful rules.

**Comment by hattrill** [comment]
For the other point, this is what I see for the node:

![Screenshot 2019-07-11 at 08 45 51](https://user-images.githubusercontent.com/11865746/61032368-37aef180-a3b9-11e9-8a2c-1b183704c4b2.png)

and these are the annotations I see from PAINT for FBgn0029830:
![Screenshot 2019-07-11 at 08 53 54](https://user-images.githubusercontent.com/11865746/61032512-7b096000-a3b9-11e9-9263-c00b37b7c4da.png)

This doesn't give me a good sense of what it does:
cell-cell adhesion and receptor localization to synapse?
